packages/terra-alert/src/Alert.scss

```diff
@@ -0,0 +1,140 @@
+@import './variables';
+
+:local {
+  /* stylelint-disable selector-class-pattern  */
+  /* stylelint-disable  property-no-unknown */
+  .alert-base {
+    align-items: stretch;
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+    overflow: hidden;
+    width: 100%;
+  }
+
+  .narrow {
+    flex-direction: column;
+  }
+
+  .wide {
+    flex-direction: row;
+  }
+
+  .icon {
+    flex-shrink: 0;
+    font-size: var(--terra-alert-icon-font-size);
+    margin-right: 0.5rem;
+  }
+
+  .body {
+    align-items: flex-start;
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: row;
+    margin-left: 0.6667rem;
+    margin-top: 1rem;
+    overflow: hidden;
+    padding-top: 0.1rem;
+  }
+
+  .body-narrow {
+    composes: body;
+    margin-bottom: 0;
+    margin-right: 0.6667rem;
+  }
+
+  .body-std {
+    composes: body;
+    margin-bottom: 1rem;
+    margin-right: 0;
+  }
+
+  .section {
+    flex: 1;
+    padding-top: 0.2rem;
+  }
+
+  .section-custom {
+    color: var(--terra-alert-custom-text-color);
+    composes: section;
+  }
+
+  .title {
+    float: left;
+    margin-right: 0.5833rem;
+  }
+
+  .actions {
+    align-items: flex-start;
+    display: flex;
+    flex-direction: row;
+    flex-shrink: 0;
+    justify-content: flex-end;
+    margin: 0.5833rem 0.5833rem 0.5833rem 0.8333rem;
```

Comment:
Style edits to fix icon/text/button alignment:
    margin: 0.4286rem 0.5833rem 0.4286rem 0.8333rem;